lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -346,4 +343,43 @@ def air_loop_hvac_apply_minimum_vav_damper_positions(air_loop_hvac, has_ddc = tr
 
     return true
   end
+
+  # Determine the fan power limitation pressure drop adjustment
+  # Per Table 6.5.3.1-2 (90.1-2019)
+  #
+  # @param air_loop_hvac [OpenStudio::Model::AirLoopHVAC] air loop
+  # @return [Double] fan power limitation pressure drop adjustment, in units of horsepower
+  def air_loop_hvac_fan_power_limitation_pressure_drop_adjustment_brake_horsepower(air_loop_hvac)
+    # Calculate Fan Power Limitation Pressure Drop Adjustment
+    fan_pwr_adjustment_bhp = 0
+
+    # Retrieve climate zone
+    climate_zone = air_loop_hvac.model.getClimateZones.getClimateZone(0)
+
+    # Check if energy recovery is required
+    is_energy_recovery_required = air_loop_hvac_energy_recovery_ventilator_required?(air_loop_hvac, climate_zone)
```

Comment:
This is used to not account for energy recovery fan credits if the baseline design does not have energy recovery.